trin-metrics/src/labels.rs

```diff
@@ -118,16 +121,16 @@ pub enum MessageLabel {
     Accept,
 }
 
-impl From<&ProtocolId> for ProtocolLabel {
-    fn from(protocol: &ProtocolId) -> Self {
-        match protocol {
-            ProtocolId::State => Self::State,
-            ProtocolId::VerkleState => Self::VerkleState,
-            ProtocolId::History => Self::History,
-            ProtocolId::TransactionGossip => Self::TransactionGossip,
-            ProtocolId::CanonicalIndices => Self::CanonicalIndices,
-            ProtocolId::Beacon => Self::Beacon,
-            ProtocolId::Utp => Self::Utp,
+impl From<&Subnetwork> for ProtocolLabel {
```

Comment:
nit: Looking at `ProtocolLabel`, I think it's not used and we can remove it. Even if it is used, I don't see why we can't use `Subnetwork` instead.